### PR TITLE
remove deprecated calls

### LINKED ISSUE #597: Don't use deprecated functions in `near-api-js`
we are still using deprecated versions if `contract.methodName` and `account.signAndSendTransaction` in `near-api-js` code base

### PR DESCRIPTION
**Pull request by mehtaphysical**
Fixes #597

### CODE REVIEW

File: test/account.test.js [hunk 1]
```diff
@@ -185,14 +196,17 @@ describe('with deploy contract', () => {
...
         expect(result2).toEqual(setCallValue);
         expect(await contract.getValue()).toEqual(setCallValue);
     });
 
     test('make function calls via contract with gas', async() => {
         const setCallValue = testUtils.generateUniqueString('setCallPrefix');
-        const result2 = await contract.setValue({ value: setCallValue }, 1000000 * 1000000);
+        const result2 = await contract.setValue({
+            args: { value: setCallValue },
+            gas: 1000000 * 1000000
```
Comment (thisisjoshford) [inline test/account.test.js:208]: Curious why we are doing a calculation here rather than just passing the actual amount.

**Review by thisisjoshford — APPROVED** [review]
Tests are passing and everything LGTM. Left one comment on the `account.test.js` file.

File: test/account.test.js (hunk 1 above)
Comment (volovyk-s) [reply test/account.test.js:208]: It's easier to count zeros this way!
It's even more useful when we are talking about time. Like, `24*60*60` is better than `86400`.

File: test/promise.test.js [hunk 2]
```diff
@@ -63,17 +68,22 @@ describe('with promises', () => {
...
-            args: null,
-            gas: '3000000000000',
-            balance: '0',
-            callback: 'callbackWithName',
-            callbackArgs: null,
-            callbackBalance: '0',
-            callbackGas: '2000000000000', 
-        }}, CONTRACT_CALL_GAS);
+        const realResult = await contract.callPromise({
+            args: {
```
Comment (volovyk-s) [inline test/promise.test.js:72]: `args: { args: {` does not look good :/ but let's not block this change

**Review by volovyk-s — APPROVED** [review]

**Pull request merged**